ui/widgets/nuxeo-card.js

```diff
@@ -184,7 +184,14 @@ import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
     _toggle() {
       if (this.collapsible) {
         this.opened = !this.opened;
-        this.notifyResize();
+        const ironCollapse = this.$$('iron-collapse');
+        const ironResize = new CustomEvent('iron-resize', { bubbles: true, composed: true });
```

Comment:
Since it is only used inside the _if_ statement, it should be declared there.
And probably you can save a variable by dispatching the event directly.

Reply:
Done.